qds_sdk/account.py

```diff
@@ -43,6 +43,15 @@ def parsers():
             choices=["true", "false"], default="false",
             help="Use previous account plan, default: false")
         create.set_defaults(func=AccountCmdLine.create)
+
+        #branding
+        branding = subparsers.add_parser(
```

Comment:
3. I checked the controller, since even this one uses link-url, we are fine. 

Will merge the changes today.